### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by rchemartin**
Add contact actions (add, edit, remove)
Give some documentation about usage

### CODE REVIEW

File: src/ActiveCampaign.php [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+    public function getBaseUrl() {
+
+        $endpointDetail = "";
+
+        // Check if it's a reseller URL
+        if (!preg_match("/https:\/\/www.activecampaign.com/", $this->url)) {
+            $endpointDetail = "/admin";
+        }
+
+        return sprintf('%s%s/api.php?api_key=%s', $this->url, $endpointDetail, $this->apiKey);
```
Comment (maxhelias) [inline src/ActiveCampaign.php]: `$endpointDetail ?: ''` and remove the line 99

File: src/ActiveCampaign.php [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+                $this->finalUrl,
+                [
+                    'form_params' => $data
+                ]
+            )
+        );
+    }
+
+    public function callGET($data = null) {
+        var_dump($data);
```
Comment (maxhelias) [inline src/ActiveCampaign.php]: No `var_dump` please

File: src/ActiveCampaign.php [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+        $responseContent = json_decode($response->getBody()->getContents());
+
+        if ($responseContent->result_code !== 1) {
+            throw new ActiveCampaignException($responseContent->result_message);
+        }
+
+        return $responseContent;
+    }
+
+}
```
Comment (maxhelias) [inline src/ActiveCampaign.php]: Add end line please and on all other files

File: src/Contact.php [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+    }
+
+    /**
+     * function buildUrl
+     *
+     * Builds the final url to your endpoint
+     *
+     * @return mixed|void
+     */
+    public function buildUrl() {
```
Comment (maxhelias) [inline src/Contact.php]: You can move this function in your `ActiveCampaign` class and remove this one

File: src/ActiveCampaign.php (hunk 1 above)
Comment (rchemartin) [reply src/ActiveCampaign.php]: Done

File: src/ActiveCampaign.php [hunk 5]
```diff
@@ -88,51 +90,56 @@ public function __construct(string $url = null, string $apiKey = null, Client $h
...
-    public function buildUrl() {
-        $this->finalUrl = sprintf('%s&api_action=%s_%s&api_output=%s', $this->url, $this->method, $this->action, self::API_OUTPUT);
+    public function buildUrl()
+    {
+        $this->finalUrl = \sprintf('%s&api_action=%s_%s&api_output=%s', $this->url, $this->method, $this->action, self::API_OUTPUT);
     }
 
     /**
-     * function callPOST
+     * function callPOST.
```
Comment (maxhelias) [inline src/ActiveCampaign.php]: Function renamed

**Pull request merged**